test_expressions/cudd_reference_outputs/many_variables_analysis.md

```
@@ -0,0 +1,157 @@
+# BDD Analysis Report
+
+## Original Expression
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)